cmake/FindLAPACK.cmake

```diff
@@ -62,9 +65,14 @@ mark_as_advanced(LAPACK_LIBRARY)
 
 _lapack_check_is_working()
 
+# TODO: Checks?
+mark_as_advanced(LAPACK_INCLUDE_DIR)
```

Comment:
Is there any reasonable check we can do on this? I think not, but I may be wrong.